Sources/ProjectSpec/FileType.swift

```diff
@@ -110,5 +110,6 @@ extension FileType {
         "xcfilelist": FileType(buildPhase: BuildPhaseSpec.none),
         "apns": FileType(buildPhase: BuildPhaseSpec.none),
         "pch": FileType(buildPhase: BuildPhaseSpec.none),
+        "storekit": FileType(buildPhase: BuildPhaseSpec.none),
```

Comment:
👋 @jcolicchio, please can you help me understand this addition quickly? 

When using a **.storekit** file in a scheme I think maybe makes sense that this wouldn't be included as a resource, however when updating to 2.19.0 today, my tested started failing because `SKTestSession(configurationFileNamed: "StoreKitConfiguration")` couldn't find my configuration file anymore. 

Could this have maybe been an oversight because to use the **.storekit** files with `SKTestSession` then they need to be bundled with the test target? 

Or is it that Xcode doesn't treat **.storekit** files as resources by default? I haven't actually tested adding one to Xcode manually to see how it behaves out of the box so maybe the prior behaviour was a side effect that I had been relying on. 

I can solve it in my spec file by overriding the `buildRule` in `sources`, but I'm wondering if maybe we want to refer this or thing actually 🤔

Reply:
Hey @liamnichols , sorry to hear my recent change has broken your setup! I'll look into how Xcode typically deals with *.storekit* files, it may indeed make sense for them to be copied by default. My use case involved referencing a storekit file in the scheme, which doesn't appear to require the *.storekit* file be copied. Apologies!

Reply:
No stress, thanks for answering my question so quickly 🙌 

TBH I was also only just finding out that **.storekit** files could be used in two separate ways 😅 I only noticed that our project was doing this because our tests started failing when I upgraded XcodeGen 😬

Reply:
Hah all good, I guess we both learned something today :P